IPs/RSKIP_tbd.md

```diff
@@ -0,0 +1,282 @@
+
+** :warning: TODO: HACER CALCULOS REALES**
+
+Possible calculation:
+```
+NUMBER_OF_SIGNATORIES => NS
+(NS * 73 + 600) * FEE_PER_BYTE
+
+Given 9 signatories and a feePerKb of 30_000 satoshis:
+37710 satoshis are to be sent to the new Powpeg
+```
+
+Once the transaction is created, it has to be stored in the set for peg-outs waiting for confirmations (i.e. "releaseTransactionSet").
+At the same time a new value in the storage should be added to identify this special kind of transaction.
+
+** :warning: TODO: DEBERIA CONSIDERAR MULTIPLES REGISTROS?**
+Suggested implementation:
+```
+KEY: "SVP_FUNDING_TX"
+VALUE: sighash of SVP funding tx
+```
+
+##### Funding transaction confirmation
+
+The SVP funding transaction goes through the usual peg-out confirmation process. Once the transaction accumulates at least 4_000 Rootstock blocks (for mainnet) it should be ready to be moved from the current set to the waiting for signatures set (i.e. "rskTxsWaitingFS").
+
+##### Funding transaction signing
+
+The funding transaction should be signed as any other peg-out. Once it is fully signed it should be removed from the set and an release_btc event should be emitted.
+From now on, the funding transaction should be broadcasted to Bitcoin, and the new Powpeg will be responsible for registering in the Bridge, and create the Proof.
+
+### Proof transaction creation
+
+For the new Powpeg to be able to proof its spendability, the funding needs to be registered, and once registered the Proof will be created, confirmed, signed and broadcasted, for the current Powpeg to validate it.
+
+#### Funding transaction registration
+
+The funding registration should be registered through a new Bridge method, dedicated specifically for SVP operations.
+
+New Bridge method signature:
+```
+registerSVPBitcoinTransaction(rawBtcTx bytes, blockHeight uint256, partialMerkleTree bytes) returns uint8
+```
+
+This method will be publicly callable, but only from EOA addresses.
+It will perform a number of verifications:
+- SVP period is still enabled (i.e. CURRENT_RSK_BLOCK - NEW_POWPEG_CREATION_BLOCK < SVP_PERIOD) ** :warning: TODO: VALIDAR!!!**
+- Transaction hash is not already registered in the Bridge (through accessing storage key "btcTxHashesAP" as already done in `registerBtcTransaction` method)
+- Block height is registered and has at least 6 confirmations (notice that we intentionally don't propose using the minimum confirmations of a regular Bitcoin transaction)
+- Transaction hash is part of provided partial merkle tree
+- Partial merkle tree is well formed and its merkle root matches the merkle root of the provided block height
+- Transaction sighash of first input is registered in the new storage (key SVP_FUNDING_TX, see [here](#funding-transaction-creation) for more details)
```

Comment:
We should discuss if every member of the new powpeg should sign this tx or if with the threshold is enough